cli/azd/pkg/apphost/generate.go

```diff
@@ -350,7 +355,7 @@ func (b *infraGenerator) LoadManifest(m *Manifest) error {
 			// resource type.
 		case "postgres.database.v0":
 			b.addContainerAppService(name, "postgres")
-		case "postgres.connection.v0", "rabbitmq.connection.v0", "azure.cosmosdb.connection.v0":
+		case "postgres.connection.v0", "rabbitmq.connection.v0", "azure.cosmosdb.connection.v0", "sqlserver.connection.v0":
```

Comment:
@mitchdenny @davidfowl - Just checking - we no longer need the `.connection.v0` types right?  I know `azd` continued to support them because we wanted to have some time for folks to migrate away from them when they were removed in Preview 1.

I'm thinking we don't need these types and so we both don't need to add this clause to our case statement and in fact could go delete most of this code now since they are no longer around and folks had time to migrate away during preview 2?

Reply:
That's correct. We can remove the connection resources now.